transaction/tests/AccessMode_unittest.cpp

```diff
@@ -0,0 +1,112 @@
+/**
+ *   Copyright 2016, Quickstep Research Group, Computer Sciences Department,
+ *     University of Wisconsin—Madison.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ **/
+
+#include "transaction/AccessMode.hpp"
+
+#include "gtest/gtest.h"
+
+namespace quickstep {
+namespace transaction {
+
+TEST(AccessModeTest, ModeCompatibilty) {
+  const AccessMode is_mode = AccessMode(AccessModeType::kIsLock);
+  const AccessMode ix_mode = AccessMode(AccessModeType::kIxLock);
+  const AccessMode s_mode = AccessMode(AccessModeType::kSLock);
+  const AccessMode x_mode = AccessMode(AccessModeType::kXLock);
```

Comment:
I removed them, because I did not use them in the lock manager. However your are right, since we added them in public. I will add them, thanks for the warning.